perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1237,7 +1237,7 @@ sub setup_garden_sub_params : Private {
         $service_id = $c->cobrand->garden_service_id;
     }
     $c->set_param('service_id', $service_id);
-    $c->set_param('client_reference', 'GGW' . $c->stash->{property}->{uprn});
+    $c->set_param('client_reference', $c->cobrand->garden_waste_subscription_client_ref);
```

Comment:
I think we can drop this line. It is not used, and is therefore confusing if people think it is - no contact has it as a parameter, so open311-adapter generates its own from the report ID (which is good because otherwise I think there would have been uniqueness issues).